packages/shared/src/components/checklist/ChecklistBar.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { ChecklistBarProps } from '../../lib/checklist';
+import { useChecklist } from '../../hooks/useChecklist';
+import { RankConfetti } from '../../svg/RankConfetti';
+
+export const ChecklistBar = ({
+  className,
+  title,
+  steps,
+}: ChecklistBarProps): ReactElement => {
+  const { isDone, completedSteps, nextStep } = useChecklist({
+    steps,
+  });
+
+  return (
+    <div className={classNames(className, 'w-full')}>
+      <div
+        className={classNames(
+          'relative flex flex-wrap gap-2 overflow-hidden bg-gradient-to-l from-raw-cabbage-90 to-raw-cabbage-50 px-2 pb-3.5 pt-3 tablet:gap-3 tablet:px-5',
+        )}
+      >
+        <div className="absolute bottom-0 left-0 right-0 flex">
+          {steps.map((step) => {
```

Comment:
We do have a ProgressBar component that can replicate similar behaviour if we give it the nr of steps. Not sure if u've considered using that

Reply:
I'll take a look if it fits my case.

Reply:
It does not really fit it because checklist has a stepped bars and `ProgressBar` is a single line, so in regards to this I would leave it as is for now if thats ok?

Reply:
<img width="351" alt="Screenshot 2024-06-13 at 11 46 29 AM" src="https://github.com/dailydotdev/apps/assets/36197304/dc1184b6-5428-456b-b42b-b09b055f978c">
Ah, looked like a line on the design, but if it's different or close to the progress the checklist has now then all good